### DISCUSSION TITLE
running on FreeBSD-amd64 ?

### DISCUSSION THREAD

**Discussion by tebra-jl** [grafana/grafana-csv-datasource#179] [Ideas]
Hi all,
My Grafana 7.5.6 running under FreeBSD-amd64
Installing this plugin with `grafana-cli plugins install marcusolsson-csv-datasource` with no error then rebbot.
By adding a new datasource with this plugin there is a error "Plugin unavailable".

**Comment by marcusolsson** [comment]
I don't believe the Grafana SDK generates binaries for FreeBSD. Ideally this would be added to the SDK, but if there's enough people who want this, I could also add another rule for this plugin specifically.
